### DISCUSSION TITLE
ask for help about the distribution of boundary planes

### DISCUSSION THREAD

**Discussion by 419278981** [mtex-toolbox/mtex#629] [Ask Anything]
How to plot this kind of MRD？
Just like this：
![QQ截图20201214135443](https://user-images.githubusercontent.com/32330722/102045444-12c37700-3e14-11eb-9fe3-fea9bf167b6b.png)
the doi：10.1016/j.actamat.2009.04.002
In order to achieve this effect, my program is as follows：
`% import_wizard
clc
clear all

ebsdN=ebsd('Iron fcc')%Iron fcc；Ni-superalloy
% grain segementation
[grains,ebsdN.grainId] = calcGrains(ebsdN('indexed'));
grains = smooth(grains,2);
ebsdN(grains(grains.grainSize <= 3)) = [];
[grains,ebsdN.grainId] = calcGrains(ebsdN('indexed'));
grains = smooth(grains,2);% grain smoothing

%define the GB
gB = grains.boundary('Iron fcc','Iron fcc')
gB3 = gB(angle(gB.misorientation,CSL(3,ebsdN.CS)) < (15*(3).^(-5/6))*degree);
gB9 = gB(angle(gB.misorientation,CSL(9,ebsdN.CS)) < (15*(9).^(-5/6))*degree);
gB27 = gB(gB.isTwinning(CSL(27,ebsdN.CS),(15*(27).^(-5/6))*degree));

% clear g ori1 ori2 distri mori axCrystal axSpecimen
g=gB3;
for i=1:size(g)
    ori1(i)=ebsdN.orientations(ebsdN.id==g.ebsdId(i,1));
    ori2(i)=ebsdN.orientations(ebsdN.id==g.ebsdId(i,2));
    distri(i)=angle(ori1(i), ori2(i)) ./ degree
    mori(i)=inv(ori1(i)) * ori2(i);
end

% IPF
axCrystal = mori.axis;

plot(axCrystal,'contourf','fundamentalRegion','antipodal','figSize','small')
mtexColorbar('title','distribution of misorientation axes in mrd')

% PF
axSpecimen = ori1 .* mori.axis('noSymmetry');

plot(axSpecimen,'contourf','fundamentalRegion','antipodal','halfwidth',2.5*degree)
mtexColorbar('title','distribution of misorientation axes in mrd')`

But the final result is very poor, thank you for your help here

**Comment by kilir** [comment]
Hi, 
for the grain boundary plane distribution, you need the information about the inclination of the boundary with respect to the surface you accquired your ebsd map on. You usually won't get that from conventional 2d ebsd unless you make some very specific assumptions. Do you have this information?
Cheers,
Rüdiger

**Reply by 419278981** [reply]
Hi
Do you mean 70° tilt angle?

**Comment by lfgmorales** [comment]
Well, unless all your grain boundaries have 70 degrees inclination (which I would say it is unlikely), i guess the 70 degrees you are talking about is the sample inclination to do the ebsd maps, correct? Do you have 3d ebsd maps? Maybe you can start with something like

https://www-user.tu-chemnitz.de/~rahi/mtexWorkshop17/presentations/David_grainboundaries_2017.pdf


http://scholar.google.ch/scholar_url?url=http://mimp.materials.cmu.edu/rohrer/papers/2015_13.pdf&hl=en&sa=X&ei=tCHXX9WALMeQmgHykKiYAg&scisig=AAGBfm0TmAmY0ECQaaJzvv10AQszMmi5jg&nossl=1&oi=scholarr




Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 14 Dec 2020, at 08:05, 419278981 <notifications@github.com> wrote:

﻿

Hi
Do you mean 70° tilt angle?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/629#discussioncomment-203869>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN732VD6JKMU6775EG3SUW2KRANCNFSM4U2GOKCQ>.

[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/629#discussioncomment-203869", "url": "https://github.com/mtex-toolbox/mtex/discussions/629#discussioncomment-203869", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by 419278981** [reply]
hello
Sorry to bother you again. Thank you for the information you provided earlier. I have a general understanding of the five-parameter grain boundary. Now I am thinking about whether I can reconstruct MRD from a single plane section by referring to this article, “_Measuring the Five-Parameter Grain-Boundary Distribution from Observations of Planar Sections_”
